src/memory/AmdFlash.hh

```diff
@@ -198,10 +201,11 @@ private:
 	const std::span<const SectorInfo> sectorInfo;
 	const power_of_two<size_t> sz;
 
-	static constexpr unsigned MAX_CMD_SIZE = 8;
+	static constexpr unsigned MAX_CMD_SIZE = 37;
 	std::array<AmdCmd, MAX_CMD_SIZE> cmd;
```

Comment:
Hmm, I'm not sure what this does with savestates. More specifically I mean loading an old savestate which has `cmd` an array of size 8, in a newer openMSX version which expects `cmd` to be an array of size 37.

Probably this should be handled via bw-compat code in `serialize()`.

Reply:
I’m planning to change it to:

    static_vector<AddressValue, 5 + 32> cmd;  // longest command is BufferProgram

After I make `static_vector` work with the serialisation, going forward it should no longer require a new serialisation version when the capacity is increased.

Also we won’t need to track a separate `cmdIdx` (which was a misnomer for `cmdSize`).

Reply:
To serialize a `static_vector` you probably need to create a specialization `serialize_as_collection<static_vector<T, N>>` in `serialize_stl.hh`.